tests/periph/flashpage/main.c

```diff
@@ -229,15 +231,27 @@ static int cmd_write_raw(int argc, char **argv)
     addr = getaddr(argv[1]);
 #endif
     /* try to align */
-    memcpy(raw_buf, argv[2], strlen(argv[2]));
+    int len;
+    if ((len = strlen(argv[2])) % 2 || (unsigned)len > sizeof(raw_buf) * 2) {
```

Comment:
Because every 2 string characters are converted to their byte represnetation.
Imagine `sizeof(raw_buf)` was 1. So the length of the input string must not be longer than 2.